src/Directory.Build.props

```diff
@@ -27,7 +27,8 @@
     <Platforms>AnyCPU;x64</Platforms>
 
     <!-- Defaults for target frameworks and architecture -->
-    <LibraryTargetFrameworks>$(FullFrameworkTFM);netstandard2.0</LibraryTargetFrameworks>
+    <LibraryTargetFrameworks>$(FullFrameworkTFM);net6.0</LibraryTargetFrameworks>
+    <LibraryTargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">net6.0</LibraryTargetFrameworks>
```

Comment:
I expected to see `netstandard2.0` here, since it should be in all of our packages. Is it better to reference it individually? Why?

Reply:
I was under the impression that we only needed M.B.Framework & Utilities to generate ref assemblies for the RoslynCodeTaskFactory.  

Remembering https://github.com/dotnet/msbuild/issues/6032#issuecomment-1011347015 now. All assemblies to ship to that `ref/` folder, not just M.B.F and U. I was treating them as separate problems. thanks for catching this one

Reply:
https://github.com/dotnet/msbuild/issues/6032#issuecomment-886212763

There are folks who need ns2.0 from our other projects. I'm not sure about the other packages, but this seems like the right move for now.